_posts/cas/2017-07-05-cas-contribution-guide.md

```diff
@@ -108,6 +110,14 @@ In order to successfully finish this exercise you need:
 2. [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), eclipse or NetBeans (Depending on the change, `vim` may be perfectly fine too)
 3. [Java (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 
+## File Contributor License Agreements
```

Comment:
Let me understand this: doesn't this clause only apply to those who are committers? Contributors should be OK to submit patches without acceptance?

Reply:
Ultimately we'd need the Board to change the formal policy. [Some](https://groups.google.com/a/apereo.org/d/msg/open/pO4HXcq8DYQ/RKn5OO0FAgAJ) Board seats are [available for re-nomination over the next couple of days](https://groups.google.com/a/apereo.org/d/msg/announcements/rCtIZGF2eos/fgkcghOgCQAJ), so this is an opportunity to move the balance a bit towards Board members who find e.g. [Ben Balter's perspective](https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/) compelling. Prevailing upon Apereo to [seat the "individual member" Board representative](https://groups.google.com/a/apereo.org/d/msg/open/pO4HXcq8DYQ/jzLDECkJAgAJ) might also help, since I'd imagine a representative of individual members would tend to be sympathetic to relieving burdens on committers and contributors.